trailblazer/store/api.py

```diff
@@ -344,6 +350,15 @@ def update_ongoing_analyses(self, ssh: bool = False) -> None:
                     f"Failed to update {analysis_obj.family} - {analysis_obj.id}: {e.__class__.__name__}"
                 )
 
+    def get_elapsed_time(self, analysis_obj: models.Analysis) -> str:
```

Comment:
Nice! We are in a hurry here, but it would be great in the future to write a unit test whenever creating a new def

Reply:
This one used to be in the comment - but as Emma asked to remove it, it is moved to inactive code block. In the future we will have a new field to keep this value